XWBTool/xwbtool.cpp

```diff
@@ -14,23 +14,7 @@
 // http://go.microsoft.com/fwlink/?LinkId=248929
 //--------------------------------------------------------------------------------------
 
-#pragma warning(push)
```

Comment:
I don't use pch.h for xwbtool in the VCXPROJ because it's only a single file. It also does not use the same set of headers as the DirectXTK library...

Reply:
As `WAVFileReader.cpp` is part of the project and that includes `pch.h`, it's necessary to make all the implementation files do the same thing.  I don't see much of an alternative; you cannot put conditional code into `WAVFileReader.cpp` as `#include "pch.h"` has to be the first statement for pre-compiled headers to work.

Reply:
You could revert the changes to xwbtool and use ``/FIpch.h``.

Reply:
Yeah that works.  Please see updated commit.